doc/main.html

```diff
@@ -148,50 +150,103 @@ <h4>Overall structure</h4>
+              <td><a>Normative references</a></td>
+              <td>Mandatory</td>
             <tr>
-              <td>0..1</td>
-              <td><a>terms and definitions section</a></td>
+              <td>4</td>
+              <td><a>Terms and definitions</a></td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..*</td>
-              <td><a>prose section</a></td>
+              <td>5-*</td>
+              <td>* (<a>Prose section</a>)</td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..*</td>
-              <td><a>annex section</a></td>
+              <td>Annex A-*</td>
+              <td>* (<a>Annex section</a>)</td>
+              <td>Optional</td>
             </tr>
             <tr>
-              <td>0..1</td>
-              <td><a>bibliography section</a></td>
+              <td>Unnumbered</td>
+              <td><a>Bibliography</a></td>
+              <td>Conditional (if informative references present)</td>
             </tr>
           </tbody>
         </table>
 
+        <p class="note">For further section infomation and requirements outside of this document, see <a href="#bib-iso-directives-part2"></a> and <a href="#bib-smpte-ag-16"></a></p>
+
+      </section>
+
+      <section id="sec-toc-section">
+        <h4>Table of contents section</h4>
+
+        <p>The <dfn>Table of contents</dfn> contains a listing of all sections within the document and links to those sections.</p>
+
+        <p>No user text is required or allowed.</p>
+
+        <p>The heading and entire section will be generated automatically.</p>
+
+      </section>
+
+      <section id="sec-foreward-section">
+        <h4>Forward section</h4>
+
+        <p>The <dfn>Foreward</dfn> contains background info about SMPTE, copyright notices, and additional infomation based on the document type (<code>pubType</code>).</p>
+
+        <p>User text for additional infomation relevant to the document is allowed in addition to the general boilerplate. If adding additional text, the section shall be present as:</p>
+
+<pre>&lt;section id=&quot;sec-foreward&quot;&gt;
+  &lt;p&gt;This is the additional infomation relevant to the document.&lt;/p&gt;
+&lt;/section&gt;
+</pre>
+        <p>The <code>id</code> attribute shall be present and equal to <code>sec-foreward</code></p>
+
+        <p>If no additional user text is required, the section shall not be present.</p>
+
+        <p>The heading and boilerplate of the section will be generated automatically.</p>
+
       </section>
 
       <section id="sec-introduction-section">
-        <h4>introduction section</h4>
+        <h4>Introduction section</h4>
+
+        <p>The <dfn>Introduction</dfn> contains the introduction of the document.</p>
 
-        <p>The <dfn>introduction section</dfn> contains the introduction of the document.</p>
+          <p>This section is optional. If using this section, it shall be present as: </p>
```

Comment:
```suggestion
          <p>Below is an example:</p>
```

I do not think we can say "shall be" since we might have multiple `<p>`, some `<ul>`, etc.

The table above already states the section is  optional, so no need to repeat it here.

Reply:
Agreed on the "optional" and example. Others sections I have update present to formatted where applicable.